packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1440,7 +1473,7 @@ export type ChatAdapterUiState = {
 };
 
 // @public
-export type ChatAttachmentType = 'unknown' | 'image';
+export type ChatAttachmentType = 'unknown' | 'image' | /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */ 'file';
```

Comment:
This seems like it shouldn't be here?

Reply:
it's still under CC flag, similar to this
`// @public
export type CallAdapterLocator = TeamsMeetingLinkLocator | GroupCallLocator | RoomCallLocator | /* @conditional-compile-remove(meeting-id) */ TeamsMeetingIdLocator;`

Reply:
This should be stripped - i.e. no comment in the API flag. It's a bit strange to have comments like this in the TypeScript definitions publicly.
The stable API should only have stabilized features, which in this case includes the 'file' addition. It looks like the CC step is not stripping the comments out in this case.

Reply:
Craeted 3745338 to track the issue of cc comments to be fixed